exercises/allergies/src/example.c

```diff
@@ -0,0 +1,32 @@
+#include "allergies.h"
+#include <stdlib.h>
+
+static const int scores[] = {
+   1,
+   2,
+   4,
+   8,
+   16,
+   32,
+   64,
+   128
+};
+
+bool is_allergic_to(Allergen_t allergen, int score)
+{
+   return ((score & scores[allergen]) == scores[allergen]);
+}
+
+void get_allergens(int score, Allergen_List_t * list)
+{
+   list->allergens = calloc(0, sizeof(Allergen_t) * Allergen_Last);
```

Comment:
`Allergen_Last` is 7, but there are actually 8 allergens.

I think it will make things a bit simpler if the allergen enum is rearranged like this:

```c
enum {
  Allergen_Eggs,
  Allergen_Peanuts,
  ...
  Allergen_Count
}
```

Then you can rearrange your `for` to use `allergen < Allergen_Count` as the termination condition and use `sizeof(Allergen_t) * Allergen_Count` as your allocation size.